### DISCUSSION TITLE
Report spam

### DISCUSSION THREAD

**Discussion by ryuujo1573** [community/community#15534] [Other Feature Feedback, Questions, & Ideas]
There are users posting political and offensive contents using GitHub functions, and polluting issues using GitHub Actions bot.

https://github.com/cirosantilli/china-dictatorship/issues

This should be controlled.

**Comment by cirosantilli** [comment]
There was a duplicate at: https://github.com/github/feedback/discussions/12641 but they deleted: https://web.archive.org/web/20220310124004/https://github.com/github/feedback/discussions/12641

Related:

- https://cirosantilli.com/china-dictatorship/github-report
- https://cirosantilli.com/china-dictatorship/github-repositories-with-censored-information
- https://cirosantilli.com/china-dictatorship/meant-to-be-used
- https://cirosantilli.com/china-dictatorship/github
- https://cirosantilli.com/china-dictatorship/github-gov-takedowns

**Comment by ettaboyle** [comment]
Hi @ryuujo1573 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info: 
### [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) 

- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. Also tagging @cirosantilli so you get a notification on the thread. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
